manager/main.go

```diff
@@ -52,15 +54,30 @@ func init() {
 func run(namespace string, metricsAddr string, enableLeaderElection bool,
 	enableApplicationController, enableBlueprintController, enablePlotterController, enableMotionController bool) int {
 	setupLog.Info("creating manager")
+	systemNamespaceSelector := fields.SelectorFromSet(fields.Set{"metadata.namespace": utils.GetSystemNamespace()})
+	workerNamespaceSelector := fields.SelectorFromSet(fields.Set{"metadata.namespace": "m4d-blueprints"})
+	selectorsByObject := cache.SelectorsByObject{
+		&appv1.Plotter{}:                {Field: systemNamespaceSelector},
+		&appv1.M4DModule{}:              {Field: systemNamespaceSelector},
+		&appv1.M4DStorageAccount{}:      {Field: systemNamespaceSelector},
+		&appv1.Blueprint{}:              {Field: workerNamespaceSelector},
+		&motionv1.BatchTransfer{}:       {Field: workerNamespaceSelector},
+		&motionv1.StreamTransfer{}:      {Field: workerNamespaceSelector},
+		&kbatch.Job{}:                   {Field: workerNamespaceSelector},
+		&corev1.Secret{}:                {Field: workerNamespaceSelector},
+		&corev1.Pod{}:                   {Field: workerNamespaceSelector},
+		&corev1.PersistentVolumeClaim{}: {Field: workerNamespaceSelector},
```

Comment:
The motion controllers are also using `kbatch.CronJob` and `apps.Deployment`

Reply:
Right... Actually, how about creating a new manager for motion, namespace-scoped? We would need less resources to restrict (only M4Dxxx, Plotters, Blueprints) this way. I was thinking about running two manager processes in main.go - what would it require? Different ports, leader-election, metrics?

Reply:
Added missing resources.